shared/shared_structs.go

```diff
@@ -21,9 +21,8 @@ type PutArgs struct {
 }
 
 type GetArgs struct {
```

Comment:
I would recommended using separate structs for GetArgs and PutArgs like "MasterToClientGetArgs", "MasterToClientPutArgs", "ClientToServerGetArgs", "ClientToServerPutArgs" because there are many extra fields we pass during the RPC call by using single set of structs. For example, master to client GetArgs, we don't need clientId. master to client PutArgs we don't need clientId, serverId, clientClock.  client to server GetArgs we don't need clientId.

Reply:
resolved